docs/pages/core/actions/connect.en-US.mdx

```diff
@@ -42,16 +42,19 @@ const result = await connect({
 
 Chain ID to connect.
 
-```ts {5}
-import { chain, connect } from '@wagmi/core'
+```ts {6}
+import { connect } from '@wagmi/core'
+import { optimism } from '@wagmi/core/chains'
 import { InjectedConnector } from '@wagmi/core/connectors/injected'
 
 const result = await connect({
-  chainId: chain.optimism.id,
+  chainId: optimism.id,
   connector: new InjectedConnector(),
 })
 ```
 
+Note: The above example is using the `optimism` chain from [`@wagmi/core/chains`](/core/chains#wagmicorechains).
```

Comment:
Should we make this an `type="info"` `Callout`?

Reply:
Can change! Thought it might have been too "in your face" though.

Reply:
Fair! We can keep this as is then.